ivy/functional/ivy/losses.py

```diff
@@ -53,22 +53,67 @@ def cross_entropy(
 
 
 # noinspection PyUnresolvedReferences
-def binary_cross_entropy(true, pred, epsilon=1e-7):
+def binary_cross_entropy(
+    true: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+    pred: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+    epsilon: Optional[float] = 1e-7,
+) -> Union[ivy.Array, ivy.Container]:
     """Computes the binary cross entropy loss.
 
     Parameters
     ----------
     true
-        true labels
+        input array containing true labels.
     pred
-        Predicted labels
+        input array containing Predicted labels.
     epsilon
-        small constant to add to log functions, default is 1e-7
+        a float in [0.0, 1.0] specifying the amount of smoothing when calculating the
+        loss. If epsilon is ``0``, no smoothing will be applied. Default: ``1e-7``.
 
     Returns
     -------
     ret
-        The binary cross entropy loss array.
+        The binary cross entropy between the given distributions.
+
+    """
+    """
+    Functional Examples
+    --------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([0, 1, 0, 0])
+    >>> y = ivy.array([0.2, 0.8, 0.3, 0.8])
+    >>> print(ivy.binary_cross_entropy(x, y))
+    ivy.array([0.2231, 0.2231, 0.3567, 1.6094])
+
+    >>> z = ivy.array([0.6, 0.2, 0.7, 0.3])
+    >>> print(ivy.binary_cross_entropy(x, z))
+    ivy.array([0.9163, 1.6094, 1.2040, 0.3567])
+
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([0, 1, 0, 1])
+    >>> y = ivy.native_array([0.2, 0.7, 0.2, 0.6])
+    >>> print(ivy.binary_cross_entropy(x, y))
+    ivy.array([0.2231, 0.3567, 0.2231, 0.5108])
+
+    With a mix of :code:`ivy.Array` and :code:`ivy.NativeArray` inputs:
+
+    >>> x = ivy.array([0, 0, 1, 1])
+    >>> y = ivy.native_array([0.1, 0.2, 0.8, 0.6])
+    >>> print(ivy.binary_cross_entropy(x, y))
+    ivy.array([0.1054, 0.2231, 0.2231, 0.5108])
+
+    Instance Method Examples
+    ------------------------
+
+    Using :code:`ivy.Array` instance method:
+
+    >>> x = ivy.array([1, 0, 0, 0])
+    >>> y = ivy.array([0.8, 0.2, 0.2, 0.2])
+    >>> print(ivy.binary_cross_entropy(x, y))
+    ivy.array([0.2231, 0.2231, 0.2231, 0.2231]))
```

Comment:
There is an extra parenthesis at the end here

Reply:
Hi @bhattbhuwan13, I just deleted the extra parenthesis